cli/src/plz/cli/operation.py

```diff
@@ -14,8 +16,9 @@ def __init__(self, configuration: Configuration):
         self.user = configuration.user
         self.execution_id = None
 
-    def url(self, *path_segments: str):
-        return self.prefix + '/' + '/'.join(path_segments)
+    def url(self, *path_segments: str, args: Dict[str, str]=None):
```

Comment:
Autoformat seems to put spaces around the `=` when you specify a type (but not if you don't). Mind running it so it's happy?

Reply:
You can just pass `params=` to the `requests` function instead of managing them yourself.